packages/dds/quorum/src/quorum.ts

```diff
@@ -229,7 +229,23 @@ export class Quorum extends SharedObject<IQuorumEvents> implements IQuorum {
      */
     // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
     public set(key: string, value: any): void {
-        // TODO: handle detached scenario, just auto accept basically
+        // If not attached, we basically pretend we got an ack immediately.
+        // TODO: Should we just directly store the value rather than the full simulation?
+        if (!this.isAttached()) {
+            // Queueing as a microtask to permit callers to complete their callstacks before the result of the set
+            // takes effect.  This more closely resembles the pattern in the attached state, where the ack will not
+            // be received synchronously.
+            queueMicrotask(() => {
+                this.handleIncomingSet(
+                    key,
+                    value,
+                    0 /* refSeq */,
+                    0 /* setSequenceNumber */,
+                    "detachedClient" /* clientId */,
```

Comment:
Not magic really.  I was originally going to just use `""` but figured something more explicit might be easier to debug later.  This also might go away pending the TODO above (storing the value directly rather than calling into `handleIncomingSet()`)